crates/starknet-server/tests/common/mod.rs

```diff
@@ -67,6 +83,36 @@ pub mod util {
         serde_json::from_slice(&resp_body_bytes).unwrap()
     }
 
+    /// Assumes Devnet has been run with the usual account seed and returns
+    /// the signer and address of the 0th account
+    pub fn get_predeployed_account_props() -> (LocalWallet, FieldElement) {
+        let signer = LocalWallet::from(SigningKey::from_secret_scalar(
+            FieldElement::from_hex_be(PREDEPLOYED_ACCOUNT_PRIVATE_KEY).unwrap(),
+        ));
+        let address = FieldElement::from_hex_be(PREDEPLOYED_ACCOUNT_ADDRESS).unwrap();
+        (signer, address)
+    }
+
+    /// dummy testing value
+    pub fn get_deployable_account_signer() -> LocalWallet {
+        let new_account_private_key = "0xc248668388dbe9acdfa3bc734cc2d57a";
+        starknet_rs_signers::LocalWallet::from(starknet_rs_signers::SigningKey::from_secret_scalar(
+            FieldElement::from_hex_be(new_account_private_key).unwrap(),
+        ))
+    }
+
+    /// resolve a path relative to the crates directory
+    pub fn resolve_crates_path(relative_path: &str) -> String {
```

Comment:
I assume this PR will fix this hacky way of accessing test files https://github.com/0xSpaceShard/starknet-devnet-rs/issues/124

Reply:
This helper function served as a fast way of not having to hardcode `CARGO_MANIFEST_DIR` everywhere.
Yes, it can be resolved with #124.